### ISSUE TITLE
[SUGGESTION] Launch / Debug Multiple Projects from Solution Explorer

### ISSUE THREAD

**Issue by scottkuhl** [microsoft/vscode-dotnettools#780]
### Describe the feature you'd like

It is currently possible to launch and debug multiple projects using compounds in launch.json.  It is also currently possible to run multiple debug sessions through the Solution Explorer if you run them one at a time.  But I don't see anyway to launch multiple projects for debugging at once through Solution Explorer.

What I would like is a feature similar to setting multiple startup projects in Visual Studio.

### Environment Information

- VS Code Version 1.84.2
- C# Dev Kit Version 1.0.14

**Comment by timheuer** [comment]
@scottkuhl wouldn't the `compound` type in launch.json fir your needs here? Once configured this becomes an option to choose from in the VS Code debug/run view and would persist last choice that you made.

**Comment by scottkuhl** [comment]
@timheuer yes, actually it does and that is what I am currently using.  But the documentation says this:

"[Debug with launch.json](https://code.visualstudio.com/docs/csharp/debugging#_debug-with-launchjson)
If you are using the C# Dev Kit, we recommend not using this option."

I opened this feature request because in some ways I agree.  Bouncing between the Solution Explorer and the file explorer is not a great experience.  Anything that can enhance the options through Solution Explorer is a win.

**Comment by timheuer** [comment]
Yeah fair discussion! I think compound is the way to go for now and works well within the VS Code experience.  We'll keep this open to discuss.  Our current VS mechanism isn't a portable one.

**Comment by xt0rted** [comment]
Unless I'm doing it wrong, using a compound task to launch multiple projects almost always gives me errors due to multiple builds kicking off and stepping on one another's file actions. Lots of locked files and usually the first project wins and then I have to go through and save a file in each of the other projects to get them to rebuild and launch. Having first class support for this, which hopefully alleviates the file locking issue, would be great.

**Comment by timheuer** [comment]
@xt0rted happy to take a look at your compound launch.json for some assist if you'd like
